### PR TITLE
Update subnetwork documentation

### PR DESCRIPTION
**Pull request by sammcveety**
R: @davorbonaci

### CODE REVIEW

File: runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineWorkerPoolOptions.java [hunk 1]
```diff
@@ -154,9 +154,8 @@ public String create(PipelineOptions options) {
    * workers.
    *
    * <p>Default is up to the Dataflow service. Expected format is
-   * regions/REGION/subnetworks/SUBNETWORK.
-   *
-   * <p>You may also need to specify network option.
+   * regions/REGION/subnetworks/SUBNETWORK or the fully qualified subnetwork name, beginning with
```
Comment (davorbonaci) [inline runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineWorkerPoolOptions.java]: beginning with http:// -> such as http://actual-example?

Comment (sammcveety) [inline runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineWorkerPoolOptions.java]: Done

**Comment by dhalperi** [comment]
Sorry we dropped the ball on merging -- needs a rebase now. I will merge once rebased. Thanks, @sammcveety

**Comment by davorbonaci** [comment]
I don't think we dropped the ball. GitHub mirroring / automation messed up something.

I think @sammcveety should just close.

**Pull request closed without merging**